### ISSUE TITLE
render media files

### ISSUE THREAD

**Issue by hbakhtiyor** [mozilla/send#554]
would be nice to show media files in browser

**Comment by wresuolc** [comment]
Would you expand on this idea a bit?  Where would you like to see media files?  As a preview when uploading?  Or before downloading?

**Comment by pdehaan** [comment]
Also, which type of "media files"?

- [ ] PNG / GIF / JPEG
- [ ] MOV / MP4 / AVI
- [ ] SWF
- [ ] MP3 / OGG
- [ ] DOC / DOCX

I imagine this could get very tricky since it confuses the concept of "download once then delete from server" since I would theoretically be able to see a JPEG preview, but then never need to download the file, making it available for others to view the preview until the file is deleted in 24h.

**Comment by hbakhtiyor** [comment]
yeah, before downloading or maybe after download? (not saving to local drive)

images, videos, audios, pdf, otherwise which support by browser

**Comment by sevaan** [comment]
Some survey feedback also asked if text files could be displayed instead of downloaded, like with Pastebin.

**Comment by funkydude** [comment]
Bump. Seriously need this for images to be of use.

**Comment by funkydude** [comment]
>I imagine this could get very tricky since it confuses the concept of "download once then delete from server" since I would theoretically be able to see a JPEG preview, but then never need to download the file, making it available for others to view the preview until the file is deleted in 24h.

If someone really wants to download an image, which virtually no one does, they can right click and save the image. Alternatively people sharing images can just increase the view count.
